### PR TITLE
Added table with some style to account setup, add background sections…

### PR DESCRIPTION
**Pull request by stephanieea**

### CODE REVIEW

**Comment by stephanieea** [comment]
@dylanavery720 @bekahlundy --  I don't know if going with a table inside a form is the right way to go for account setup, what do you think?  

I used the vh's to try and soften the differences between different mobile displays but it's still not as crisp or responsive as it should be- I wonder if we need to go all responsive (em or vh) or all pixels- what concerns me about using pixels is how sections won't lay flat against each other in the same way on different devices...  but switching it all seems like it might be overkill...  

Also, I left the user prop being passed to header in the application as a reminder to talk about how to display the h2 as a conditional- I was struggling with getting it to work and didn't want to forget about it- so that's just there as a reminder.

File: lib/components/Application.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ createArrayFromFB(object) {
   render() {
     return (
       <div className='application'>
-        <Header />
+        <Header user={this.state.user}/>
```
Comment (dylanavery720) [inline lib/components/Application.js:50]: Gooid idea!

File: lib/components/AccountSetup.js [hunk 2]
```diff
@@ -2,8 +2,23 @@ import React from 'react';
...
+    <form>
+      <table className='setup-table'>
+        <tbody>
+      <tr>
+        <td className='tdQuest'>How much money do you want to spend this month?</td>
+        <td><input className='tdInput'/></td>
+        <td><button className='table-btn'></button></td>
+      </tr>
+      <tr>
+        <td className='tdQuest'>How much money do you want to spend on rent?</td>
```
Comment (dylanavery720) [inline lib/components/AccountSetup.js:14]: Cool looks like a good start, will play around in here today and think about your questions.

**Review by dylanavery720 — APPROVED** [review]
Looks good! I will think about your questions as I play around with it, good idea to put the user prop in the header.

Dylan

**Pull request merged**